File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/ChatFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.Chat;

public class ChatFilterValidator extends FilterValidator<Chat> {

    public ChatFilterValidator(CustomValidator<Chat> validator) {
        super(Chat.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/ChatFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.Chat
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class ChatFilterValidator(validator: Validator<Chat>) : FilterValidator<Chat>(
    Chat::class, validator)
